### PR TITLE
Release candidate 3.2.2

### PR DESCRIPTION
**Pull request by maxulysse**
Need:

- [x] https://github.com/nf-core/sarek/issues/1095

Would be nice with

- [x] https://github.com/nf-core/sarek/issues/1094
  - [x] https://github.com/nf-core/sarek/issues/1098
- [x] https://github.com/nf-core/sarek/issues/1065
  - [x] https://github.com/nf-core/sarek/issues/1101
- [x] https://github.com/nf-core/sarek/issues/1069
  - [x] https://github.com/nf-core/sarek/issues/1100
- [x] https://github.com/nf-core/sarek/issues/1105
- [x] https://github.com/nf-core/sarek/issues/1106
- [x] https://github.com/nf-core/sarek/issues/1109

### CODE REVIEW

File: .github/workflows/pytest-workflow_release.yml [hunk 1]
```diff
@@ -0,0 +1,170 @@
+name: pytest-workflow-release
+# This workflow runs the pipeline with the minimal test dataset to check that it completes without any syntax errors
+on:
+  push:
```
Comment (maxulysse) [inline .github/workflows/pytest-workflow_release.yml:4]:
```suggestion
  pull_request:
```

**Review by FriederikeHanssen — APPROVED** [review]
not a good person to approve, but lgtm

File: .github/workflows/pytest-workflow_release.yml [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+  test:
+    name: ${{ matrix.tags }} ${{ matrix.profile }} NF ${{ matrix.NXF_VER }}
+    runs-on: ubuntu-latest
+    needs: changes
+    if: needs.changes.outputs.tags != '[]'
+    strategy:
+      fail-fast: false
+      matrix:
+        tags: ["${{ fromJson(needs.changes.outputs.tags) }}"]
+        profile: ["docker", "singularity", "conda"]
```
Comment (maxulysse) [inline .github/workflows/pytest-workflow_release.yml:37]:
```suggestion
        profile: ["docker", "singularity"]
```

File: .github/workflows/pytest-workflow_release.yml [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+          - profile: "conda"
+            tags: merge
+          - profile: "conda"
+            tags: snpeff
+          - profile: "conda"
+            tags: umi
+          - profile: "conda"
+            tags: validation_checks
+          - profile: "conda"
+            tags: vep
```
Comment (maxulysse) [inline .github/workflows/pytest-workflow_release.yml:59]:
```suggestion
          - profile: "conda"
            tags: concatenate_vcfs
          - profile: "conda"
            tags: deepvariant
          - profile: "conda"
            tags: haplotypecaller
          - profile: "conda"
            tags: merge
          - profile: "conda"
            tags: snpeff
          - profile: "conda"
            tags: umi
          - profile: "conda"
            tags: validation_checks
          - profile: "conda"
            tags: vep
```

**Review by asp8200 — APPROVED** [review]
LGTM - except for the failing tests due to exceeded API rate limit.

**Pull request merged**